functions.php

```diff
@@ -243,11 +243,11 @@ function nightingale_widgets_init() {
  * Enqueue scripts and styles.
  */
 function nightingale_scripts() {
-	wp_enqueue_style( 'nightingale-style', get_template_directory_uri() . '/style.min.css', array(), '20210721' );
-	wp_enqueue_style( 'nightingale-page-colours', get_template_directory_uri() . '/page-colours.min.css', array(), '20210721' );
```

Comment:
using time() prevents browser caching and is bad for performance (especially mobile)